device-types/Raritan/PX3-5729R.yaml

```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: Raritan
+model: PX3-5729R
+slug: px3-5729r
+part_number: PX3-5729R
+u_height: 1
+is_full_depth: false
```

Comment:
Please add the `airflow` data point if you have/can find it

Reply:
It's a PDU, so no fans/airflow information that I'm aware of